CHANGELOG.md

```diff
@@ -25,10 +16,37 @@
     - **aviatrix_spoke_gateway**
     - **aviatrix_edge_spoke_gateway**
     - **aviatrix_transit_gateway**
-5. Added new attribute ``bgp_bfd`` and ``enable_bfd`` to support bgp_bfd configuration in the following resources
+2. Added new attribute ``bgp_bfd`` and ``enable_bfd`` to support bgp_bfd configuration in the following resources.
     - **aviatrix_transit_external_device_conn**
     - **aviatrix_edge_spoke_external_device_conn**
     - **aviatrix_spoke_external_device_conn**
+3. Added/Updated Edge Terraform documentation to include interface mapping.
+4. Updated documentation for ``aviatrix_firewall_install`` where `user_data` was not applicable to Palo Alto firewalls. There is no longer
+   a restriction on using `user_data` on Palo Alto Firewalls.
+5. Updated documentation for ``aviatrix_spoke_gateway. Clarified use of the `included_advertised_spoke_routes` attribute.
+6. Updated documentation for ``aviatrix_smart_group`` with examples of external groups useage.
```

Comment:
```suggestion
6. Updated documentation for **aviatrix_smart_group** with examples of external groups usage.
```
As per the previous comments for following syntax for resources and attributes to avoid confusion

Reply:
done